source/less/blocks/image-bridge.less

```
@@ -0,0 +1,3 @@
+.image-bridge {
+  background: url("photo-bridge@1x.jpg") no-repeat;
```

```suggestion
  background: url("../img/photo-bridge@1x.jpg") no-repeat;
```
Относительный путь - он относительно чего. Это файл стилей и он находится в каталоге **css** (после сборки препроцессором). Браузер увидев такой путь `url("photo-bridge@1x.jpg")` станет искать файл в каталоге **css**, но его там нет, так как этот файл находится в каталоге **img**. Относительно каталога **css** каталог **img** находится по пути **../img/** (символы **../** указывают браузеру перейти на один уровень вверх относительно **css** (текущим каталогом станет **./**, то есть корневой каталог сайта) и там перейти в каталог **img**.